index.html

```diff
@@ -432,7 +432,8 @@
+,{"hide":false,"system":"Tablespace-rim-tuned","date":"2024-02-25","machine":"L1 - 16CPU 32GB","cluster_size":1,"comment":"","tags":["C","Rust","row-oriented","column-oriented","PostgreSQL compatible","managed","analytical"],"load_time":13711,"data_size":135469176422,"result":[[0.436542,0.032826,0.058116],[0.475804,0.06273,0.069842],[0.221264,0.118931,0.108054],[1.13058,0.110611,0.107611],[0.842296,0.840424,0.84895],[2.84863,1.22785,1.17402],[0.009753,0.002873,0.003417],[0.195597,0.098896,0.074905],[1.70796,1.37998,1.54147],[1.4772,1.33613,1.31476],[0.373194,0.340836,0.338413],[0.375037,0.361468,0.378387],[1.16898,1.22366,1.21796],[2.09798,2.94135,2.16265],[1.52902,1.34797,1.2899],[0.972571,0.981924,0.914636],[2.43485,2.58636,2.75724],[0.005437,0.003755,0.003427],[5.30082,5.16673,5.20788],[0.006782,0.002805,0.002702],[16.7202,0.179415,0.169258],[0.210038,0.180524,0.174665],[23.7864,0.2504,0.242016],[0.183456,0.173261,0.17237],[0.048444,0.003245,0.003059],[0.007258,0.003278,0.003245],[0.015761,0.003279,0.003193],[22.3888,1.79019,1.70748],[17.1564,4.59906,18.7535],[0.599159,0.572579,0.5331],[1.20788,1.13153,1.1528],[3.71611,1.39238,1.40143],[8.40708,8.20716,7.2647],[46.2087,5.43577,45.5534],[6.23327,6.50139,6.08041],[2.44898,2.23633,2.08641],[0.42763,0.352619,0.30957],[0.678261,0.476425,0.445909],[0.179801,0.154265,0.160855],[0.776884,0.791492,0.785599],[0.171634,0.11076,0.090103],[0.110994,0.084385,0.084405],[0.103362,0.10878,0.093837]],"source":"tablespace-tuned/results/large-1.json"}
+,{"hide":false,"system":"Tablespace-rim","date":"2024-02-25","machine":"L1 - 16CPU 32GB","cluster_size":1,"comment":"","tags":["C","Rust","row-oriented","column-oriented","PostgreSQL compatible","managed","analytical"],"load_time":2197,"data_size":70306468870,"result":[[278.478,280.045,280.31],[280.129,280.298,280.087],[280.183,280.198,280.32],[280.072,280.311,280.127],[305.268,303.934,303.885],[361.026,360.565,360.701],[278.873,280.297,280.06],[280.209,280.224,280.332],[337.907,333.984,334.2],[336.371,335.349,335.22],[281.725,282.435,282.364],[282.574,282.778,282.699],[315.599,315.52,315.788],[312.013,311.874,311.971],[318.948,319.252,319.038],[349.777,349.912,347.871],[355.208,357.579,357.511],[335.828,336,337.1],[376.254,376.174,377.624],[279.008,280.414,280.248],[285.469,285.249,285.646],[285.434,285.624,285.368],[280.508,280.574,280.606],[285.239,285.642,285.474],[280.33,280.445,280.605],[280.413,280.599,280.515],[280.471,280.324,280.595],[285.202,285.641,285.491],[313.373,311.882,310.575],[279.191,280.591,280.465],[294.652,293.099,293.54],[293.603,292.951,293.018],[487.299,485.86,486.417],[645.155,613.465,609.424],[585.498,580.802,581.091],[317.183,319.422,317.918],[280.888,281.093,281.046],[280.091,280.783,280.968],[280.077,280.63,280.528],[283.704,282.535,282.118],[279.219,280.598,280.513],[280.456,280.312,280.612],[280.222,280.646,280.446]],"source":"tablespace/results/large-1.json"}
```

Comment:
@tinybit This "unoptimised" benchmark doesn't make any sense. The value proposition is that Tablespace adds is adding a Columnstore index to Postgres. Its not a new database. Showing the benchmark numbers without the Columnstore index in the benchmark is not really showing anything. You're just running the benchmark on Postgres without any indexes.